libs/payments/ui/src/lib/server/components/UpgradePurchaseDetails/en.ftl

```diff
@@ -12,4 +12,5 @@ upgrade-purchase-details-prorated-upgrade = Prorated Upgrade
 upgrade-purchase-details-new-plan-daily = { $productName } (Daily)
 upgrade-purchase-details-new-plan-weekly = { $productName } (Weekly)
 upgrade-purchase-details-new-plan-monthly = { $productName } (Monthly)
+upgrade-purchase-details-new-plan-halfyearly = { $productName } (Half-yearly)
```

Comment:
If this is the same as every six months, it might be good to add a comment for localizers to provide that context as "Half-yearly" might not be a common way of phrasing this in some languages.

Reply:
Per our discussion, I've updated this string from `{ $productName } (Half-yearly)` to `{ $productName } (6-month)`.